src/style.css

```diff
@@ -0,0 +1,131 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir.ttc") format("truetype");
+  font-weight: 400;
+  font-display: swap;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Book.ttf") format("truetype");
+  font-weight: 300;
+  font-display: swap;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Heavy.ttf") format("truetype");
+  font-weight: 900;
+  font-display: swap;
+  font-style: normal;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+}
+
+.search-form {
+  width: 100%;
+  margin-top: 20px;
+}
+
+.search-wrapper {
+  position: relative;
+}
+
+.search-wrapper:before {
+  content: "";
+  position: absolute;
+  left: 26px;
+  top: 2px;
+  background-image: url(./images/Search.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: contain;
```

Comment:
you can use background shorthand
https://www.w3schools.com/css/css_background_shorthand.asp